root/etc/e-smith/templates/httpd/vhost-extra/25IndexesOptions

```diff
@@ -0,0 +1,14 @@
+{
+use esmith::ConfigDB;
+my $vdb = esmith::ConfigDB->open_ro('vhosts') || die("Can't open vhosts db");
+
+my $indexes = $vdb->get_prop("$VhostName",'Indexes') || '';
+return unless ($indexes eq 'enabled');
+
+$OUT .= << 'EOF';
+        #25IndexesOption
+        <LocationMatch "^/">
```

Comment:
I definitely agree, but it is the behavior provided by upstream configuration and already deployed. Can we change it on running systems?